src/encord_active/app/common/components/metric_summary.py

```diff
@@ -1,22 +1,113 @@
 from typing import List
 
-import numpy as np
 import pandas as pd
 import streamlit as st
 from pandera.typing import DataFrame
+from streamlit.delta_generator import DeltaGenerator
 
 from encord_active.app.common.components import build_data_tags
 from encord_active.app.common.components.data_quality_summary import summary_item
 from encord_active.app.common.components.tags.individual_tagging import multiselect_tag
-from encord_active.app.common.state import get_state
+from encord_active.app.common.state import MetricOutlierInfo, MetricsSeverity, get_state
```

Comment:
Should `MetricOutlierInfo` and `MetricsSeverity` really belong to the state class? I would probably put them somewhere else. Probably in `lib/dataset/outliers.py`.